src/recorder/jamrecorder.cpp

```diff
@@ -41,28 +41,40 @@ using namespace recorder;
  * Creates a file for the raw PCM data and sets up a QDataStream to which to write received frames.
  * The data is stored Little Endian.
  */
-CJamClient::CJamClient(const qint64 frame, const int _numChannels, const QString name, const CHostAddress address, const QDir recordBaseDir) :
-    startFrame (frame),
-    numChannels (static_cast<uint16_t>(_numChannels)),
-    name (name),
-    address (address),
-    out (nullptr)
+CJamClient::CJamClient ( const qint64       frame,
+                         const int          _numChannels,
+                         const QString      name,
+                         const CHostAddress address,
+                         const QDir         recordBaseDir ) :
+    startFrame ( frame ),
+    numChannels ( static_cast<uint16_t> ( _numChannels ) ),
+    name ( name ),
+    address ( address ),
+    out ( nullptr )
 {
     // At this point we may not have much of a name
-    QString fileName = ClientName() + "-" + QString::number(frame) + "-" + QString::number(_numChannels);
+    QString fileName = ClientName() + "-" + QString::number ( frame ) + "-" +
+                       QString::number ( _numChannels );
     QString affix = "";
-    while (recordBaseDir.exists(fileName + affix + ".wav"))
+    while ( recordBaseDir.exists ( fileName + affix + ".wav" ) )
     {
-        affix = affix.length() == 0 ? "_1" : "_" + QString::number(affix.remove(0, 1).toInt() + 1);
+        affix =
```

Comment:
See, this shouldn't be split after the `=`.  There's no need.  Having the `?` and `:` on their own lines I'm okay with.

I don't know whether there really is a project style for `?:` usage, though.

Reply:
```cpp
        affix =
            affix.length() == 0
                ? "_1"
                : "_" + QString::number ( affix.remove ( 0, 1 ).toInt() + 1 );
```
that happens because of `BreakBeforeTernaryOperators: true`
so it "has to" break both before `?` AND `:` (unless all fits into one line)
if we set `BreakBeforeTernaryOperators: false` it would look like this
```cpp
        affix = affix.length() == 0 ?
                    "_1" :
                    "_" + QString::number ( affix.remove ( 0, 1 ).toInt() + 1 );

```

anyway with `ColumnLimit: 120` that one will be one line